build-system/tasks/create-module-compatible-es5-bundle.js

```diff
@@ -31,8 +31,7 @@ const gulp = $$.help(require('gulp'));
 exports.createModuleCompatibleES5Bundle = function(src) {
   return gulp.src('dist/' + src)
       .pipe($$.sourcemaps.init({loadMaps: true}))
-      .pipe($$.rename(src.replace(/\.js$/, '-module.js')))
-      .pipe($$.regexpSourcemaps(/(global\?global:\w*})\(this\)/, '$1(self)', 'module-global'))
+      .pipe($$.regexpSourcemaps(/(window.global\?window.global:\w*)this/, '$1self', 'module-global'))
```

Comment:
just for my own knowledge how is the regexp name used (what is its usefulness)?

Reply:
@jridgewell , i wanted to ask that too

Reply:
I think it's the filename this new code will be associated with.